osversion/osversion.go

```diff
@@ -46,6 +46,12 @@ func Get() OSVersion {
 	return osv
 }
 
+// Build gets the build-number on Windows
+// The calling application must be manifested to get the correct version information.
+func Build() uint16 {
```

Comment:
FWIW; I was in doubt wether to name this `Build()` or `BuildNumber()`

Reply:
`Build` https://msdn.microsoft.com/en-us/library/windows/desktop/ms724439(v=vs.85).aspx